development/modules/students/servicesStudents.js

```diff
@@ -36,9 +37,62 @@
       return Student.logout()
     }
 
+    function CommentStudentRelation (student) {
+      this.basePath = originsManager.getOrigin() +
+        "/" + ROUTES.STUDENTS.__BASE__ +
+        "/" + student.id +
+        "/" + ROUTES.STUDENTS.COMMENTS;
+
+    }
+
+    CommentStudentRelation.prototype = new Array();
+    CommentStudentRelation.prototype.getComments = function () {
+      return $http.get(this.basePath)
+        .then(function (response) {
+          response.data.forEach(function (comment) {
+
+            comment.publication = {
+              active: false,
+              isPublicationLoaded: false
+            };
+            comment.getPublication = function () {
```

Comment:
Why you're not using NewsService if that have all you need to that the same? Please, do not reinvent the wheel, be modular.